resume.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
```

Comment:
The look and feel of this page is a bit off. You're using Serif fonts rather than the Poppins font that you're supposed to, probaly because you forgot to link the font in  this file and didn't specify a fallback font. Also, the blue highlights are missing